test/hotspot/jtreg/serviceability/dcmd/vm/SystemMapTestBase.java

```diff
@@ -103,7 +108,12 @@ private static class LinuxPatterns implements MapPatterns {
         };
 
         public String[] shouldMatchUnconditionally() {
-            return shouldMatchUnconditionally_linux;
+            if (WhiteBox.getWhiteBox().isStatic()) {
+                return shouldMatchUnconditionally_linux;
+            } else {
+                return StringArrayUtils.concat(shouldMatchUnconditionally_linux,
+                                               shouldMatchUnconditionally_linux_libjvm);
```

Comment:
Here, and in DumpTest: please provide a short comment about why we don't want to match libjvm (even if it seems obvious).

Reply:
Add comments. Please let me know if that covers all cases in your suggestion.

@tstuefe Could you please approve again after the updates? Thanks for the review!